bindata/network/openshift-sdn/alert-rules.yaml

```diff
@@ -39,40 +39,39 @@ spec:
       for: 1h
       labels:
         severity: warning
-    - alert: IPTableSyncSDNPod
+    - alert: NodeProxyApplySlow
       annotations:
-        message: SDN pod {{"{{"}} $labels.pod {{"}}"}} on node {{"{{"}} $labels.node {{"}}"}} takes too long to sync iptables rules.
+        message: SDN pod {{"{{"}} $labels.pod {{"}}"}} on node {{"{{"}} $labels.node {{"}}"}} is taking too long to apply iptables rules.
```

Comment:
The "Slow" vs "Stale" distinction in the names is good but I think the messages could still distinguish them better.

"SDN pod X on node Y is encountering iptables scaling issues that are slowing down service updates"
"SDN pod X on node Y is failing to process queued service updates"

? dunno...

Reply:
Good point. I'll tighten up the language.

Reply:
"SDN pod ... is taking too long to apply kubernetes service rules to iptables."

@juanluisvaladas any wording suggestions? This is customer-facing.

Reply:
My issue being that "is taking too long to apply iptables rules" in the current text is ambiguous as to which problem it's describing. In the "Slow" alerts, it is "taking too long to apply iptables rules" in the sense that updates are being processed, but each update takes longer to process than we think it should, while in the "Stale" alerts, it is "taking too long to apply iptables rules" in the sense that too much time has gone by with no updates being processed.

Reply:
"SDN pod ... is taking too long to apply kubernetes service rules to iptables." sounds about right to me.

Saying it's failing to process queued service updates may sound like it never works and the alert means that it probably gets there eventually.

This may lead to unnecessary panic. Customers don't usually know that much about internal components, so I would totally expect customers evacuating a bunch of nodes with the alert at the same time, making the whole cluster state much worse than it used to be.

The NodeProxyApplyStale on the other hand could probably be more clear about it being stale.